mdpow/equil.py

```diff
@@ -466,8 +467,8 @@ def _MD(self, protocol, **kwargs):
         # so instead of fuffing with GMXLIB we just dump it into the directory
         try:
             shutil.copy(config.topfiles['residuetypes.dat'], self.dirs[protocol])
-        except:
-            logger.warn("Failed to copy 'residuetypes.dat': mdrun will likely fail to write a final structure")
+        except IOError:
```

Comment:
Very good adding an error... I was young when I wrote this code and didn't know any better — today I tell my students to never do blanket exception handling.